plugins/org/src/components/Cards/OwnershipCard/useGetEntities.ts

```diff
@@ -197,11 +198,13 @@ export function useGetEntities(
     return topN.map(topOwnedEntity => ({
       counter: topOwnedEntity.count,
       type: topOwnedEntity.type,
```

Comment:
maybe it should use only `spec.type` here with such a new setup instead of 
```ts
type: ownedEntity.spec?.type?.toString() ?? ownedEntity.kind,
```

And I wonder about `name` as well. Essentially, the upper case part could be done when rendering. No need for a separate field.
```ts
name: topOwnedEntity.type.toLocaleUpperCase('en-US'),
```

Those things could be part of the presentation logic. E.g., use type as title and kind as sub-title if there is a type, otherwise use kind as title only; title in upper case.

As `type` is currently used for styles as well. Here, we could consider both `kind` and `type` (e.g. we could use different shades of the same base color for the same kind but different types -- or some other logic/...).